core/src/test/java/org/apache/calcite/test/ReflectiveSchemaTest.java

```diff
@@ -625,6 +637,12 @@ private void check(ResultSetMetaData metaData, String columnName,
   private void checkOp(CalciteAssert.AssertThat with, String fn) {
     for (Field field : EveryType.numericFields()) {
       for (Field field2 : EveryType.numericFields()) {
+        //Decimal +-*/ others types has bug see CALCITE-2861
+        //when CALCITE-2861 was solved, remove this code logic
```

Comment:
There is a util class[1] for recording unresolved bug programmatically. You may want to use that rather than doing something like this.

[1] https://github.com/apache/calcite/blob/master/core/src/main/java/org/apache/calcite/util/Bug.java

Reply:
OK